internal/controller/volsync/vshandler_test.go

```diff
@@ -792,42 +792,35 @@ var _ = Describe("VolSync_Handler", func() {
 					}, maxWait, interval).Should(Succeed())
 				})
 
-				Context("When no running pod is mounting the PVC to be protected", func() {
```

Comment:
Instead of deleting this case, I suggest inverting the expectation. Ensure that a RS is created even in this case.